src/main/java/com/google/api/codegen/discovery/transformer/js/JSSampleMethodToViewTransformer.java

```diff
@@ -0,0 +1,159 @@
+/* Copyright 2016 Google Inc
```

Comment:
2017 on new files.

Keep the creation date on old files (cf. https://engdoc.corp.google.com/eng/doc/copyright.html?cl=head)